Dockerfile

```diff
@@ -41,6 +41,9 @@ RUN if [ "$use_pre_trained_model" = "true" ] ; then \
       # rename the directory that contains the custom-trained model artifacts
       if [ -d "./custom_assets/" ] ; then \
         rm -rf ./assets && ln -s ./custom_assets ./assets ; \
```

Comment:
@MLnick @ptitzler just double checking that the Dockerfile changes are okay before we merge.

Reply:
Where does the training process pick that up from? Does it also download the vocab file?

In NET model for example, the vocab files are provided via the COS `data/assets` folder (assumed to come from the `data_prep` phase). They are then written out to the saved model folder under `assets`, so that when that is transferred to `custom_assets` the vocab is in place. see https://github.com/IBM/MAX-Named-Entity-Tagger/blob/master/training/training_code/train_ner.py#L191

Can we follow a similar approach here?

Reply:
@MLnick I've done something similar here except download the files in dockerfile so that it doesn't repeat every training run.

Reply:
The vocab file must surely be required for the training process? Not sure I understand how it cannot - since the text / words needs to be embedded no?

Reply:
Just checked through the code, it gets the vocab file from the google cloud COS during training. It also fetches the pre-trained weights from the same source.

Reply:
I've made it simpler by uploading just the vocab file. Now it can be downloaded directly when needed. 

60b92a3

Reply:
^ this is ok ...

But my main point is this: the custom model files in `custom_assets` _only_ comes about as a result of the training process. Therefore, since the training process itself downloads the vocab file from COS (and presumably loads that file in memory), then to me it makes more sense for the training script (either the `py` or `sh` scripts) to write that file out as part of the saved model directory structure. That way it will naturally be put in `custom_assets` when packaged and downloaded as part of the `wml_train` final step.

That is exactly what I did in Named Entity Tagger - then just ensure the pre-trained model and the custom model pick up the vocab file from the same location.

Does that make sense?

Reply:
@MLnick yes, that makes more sense. I have updated the training scripts (both `train-max-model.sh` and the training python file) to factor this in. Now there is no change to the Dockerfile and the vocab file is copied over from the training run. 

0162cac

Reply:
I have also tested it out on WML + rebuilding the docker container.

Reply:
If the changes look good, fell free to merge it. I have also raised an internal PR for the landing page.